github/apps_test.go

```diff
@@ -90,6 +90,7 @@ func TestAppsService_ListInstallations(t *testing.T) {
                                    "target_id":1,
                                    "target_type": "Organization",
                                    "permissions": {
+									   "actions": "read",
```

Comment:
Could you please make this line match the indentation of the following line (i.e. remove the tabs and replace with the same number of spaces in line 94)?

Reply:
hmm yeh I noticed this, formatted multiple times and no change.. thought it might be weird gh rendering?

Reply:
nevermind, fixed, sorry!